raiden/raiden_service.py

```diff
@@ -347,6 +347,10 @@ def start(self) -> None:
         )
 
         if self.wal.state_manager.current_state is None:
+            print(
+                "This is the first time Raiden is being used with this address. "
+                "Processing all the events may take some time. Please wait ..."
+            )
```

Comment:
Is this in the wrong place? because creating a chain state is not a time consuming process. You would want to add this when we're actually restoring the chain state from a state changes or a snapshot.

Reply:
What I understood from the issue is that we want to give the warning if this is the first time raiden starts with a given address since then it needs to synch with all the events from the chain. So it should be when there has been no state present before and we are about to do all the work.

Reply:
Ah ok... i thought this was about "loading the state from the DB".